mpags-cipher.cpp

```diff
@@ -0,0 +1,106 @@
+#include <iostream>
+#include <string>
+#include <cctype>
+#include <vector>
+int main(int argc, char* argv[])
+{
+    const std::vector<std::string> cmdLineArgs { argv, argv+argc }; 
+
+    long unsigned int length { cmdLineArgs.size() };
+    for (int i{1}; i < length; ++i)  // this wont print the code name itself, just the arguements 
```

Comment:
When I compiled I got a warning about this line (and also lines 30 and 43) about comparison of signed and unsigned quantities.  Indeed, since the size of a container cannot be negative, it makes sense to use unsigned quantities when iterating through the indices.  You can use the same `std::size_t` type as above.
```suggestion
    for (std::size_t i{1}; i < length; ++i)  // this wont print the code name itself, just the arguements 
```
Good that you've started the loop from 1 and good consistent use of uniform initialisation.